conda_smithy/configure_feedstock.py

```diff
@@ -73,7 +73,7 @@ def render_run_docker_build(jinja_env, forge_config, forge_dir):
                 # "recipe/yum_requirements.txt" file. After updating that file,
                 # run "conda smithy rerender" and this line be updated
                 # automatically.
-                yum install -y {}
+                /usr/bin/sudo -n yum install -y {}
```

Comment:
Should `yum` be `/usr/bin/yum`? It seems to work without it, but mainly thinking about this [comment]( https://github.com/conda-forge/docker-images/pull/36#r79469208 ).